packages/input-select/src/Select.js

```diff
@@ -28,7 +29,10 @@ class Select extends React.Component {
     super(props);
 
     this.state = {
-      selectedValue: props.value,
+      selectedValue:
+        props.multi && !Array.isArray(props.value)
```

Comment:
I wonder if we should show a warning when it's `multi` and `value` is not an array because we could keep getting `value` as a primitive type instead of an array after component mounts 🤔